cargo-dist/src/init.rs

```diff
@@ -58,7 +58,12 @@ pub fn do_init(cfg: &Config, args: &InitArgs) -> DistResult<()> {
 
         // Immediately re-exit the process with the same
         // exit code the unhandled ctrl-c would have used
-        std::process::exit(130);
+        let exitstatus = if std::env::consts::OS == "windows" {
```

Comment:
0.o i have never seen this before (more conventional would be `if cfg!(windows`) {`

Reply:
No idea why I reached for `std::env::consts::OS` here 😅